lib/tasks/sphinx.rake

```diff
@@ -23,7 +28,7 @@ namespace :sphinx do
       # As we enqueue, we only need the :id
       scope.select(:id).except(:includes).find_in_batches(batch_size: 1000) do |batch|
         batch.each do |record|
-          SphinxIndexationWorker.perform_later(record.class, record.id)
+          SphinxIndexationWorker.new(record.class, record.id).enqueue queue: BULK_INDEXING_QUEUE
```

Comment:
Do we need to add a constantin the global namspace? rubocop complains about many constants. And is later not always clear whether something external relies on that or not. 

```suggestion
          SphinxIndexationWorker.new(record.class, record.id).enqueue queue: (Rails.configuration.three_scale.bulk_indexing_queue || 'default')
```